modules/@apostrophecms/ui/ui/apos/components/AposContextMenuTip.vue

```diff
@@ -52,14 +61,14 @@ export default {
     right: 20px;
   }
 
-  [x-placement^='bottom'] > .apos-context-menu__tip {
-    top: -11px;
+  [x-placement^='bottom'] .apos-context-menu__tip {
+    top: -27px;
     bottom: auto;
   }
 
-  [x-placement^='top'] > .apos-context-menu__tip {
```

Comment:
Did you find the case where these tooltips are created inside one another? Thats why we're using the direct child selector.

Reply:
jesus. where's that?

Reply:
I don't see any cases of that. The rich text linking modal doesn't have it, which is where we suspected it might be. We could maintain this specificity with a few more levels of `> parentEl`, but we don't want to be trying to pass some direction value down to recreate the attribute.

Reply:
i think we should put the direct child selectors back for now even though we can't find the nested example .. we never want to reach down the tree to apply these rules, they are only relevant in their immediate context

Reply:
![done](https://media.giphy.com/media/ogww893cZgaME/giphy.gif)